test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ERCPrecompileSuite.java

```diff
@@ -4559,6 +4560,62 @@ private HapiSpec erc721TransferFromWithApproveForAll() {
+                                .initialSupply(10L)
+                                .maxSupply(1000L)
+                                .treasury(TOKEN_TREASURY)
+                                .adminKey(MULTI_KEY)
+                                .supplyKey(MULTI_KEY),
+                        uploadInitCode(ERC_20_CONTRACT),
+                        contractCreate(ERC_20_CONTRACT),
+                        tokenAssociate(OWNER, FUNGIBLE_TOKEN),
+                        tokenAssociate(RECIPIENT, FUNGIBLE_TOKEN),
+                        tokenAssociate(ERC_20_CONTRACT, FUNGIBLE_TOKEN),
+                        cryptoTransfer(moving(10, FUNGIBLE_TOKEN).between(TOKEN_TREASURY, OWNER)))
+                .when(
+                        withOpContext(
+                                (spec, opLog) ->
+                                        allRunFor(
+                                                spec,
+                                                contractCall(
+                                                                ERC_20_CONTRACT,
+                                                                TRANSFER_FROM,
+                                                                HapiParserUtil.asHeadlongAddress(
+                                                                        asAddress(
+                                                                                spec.registry()
+                                                                                        .getTokenID(
+                                                                                                FUNGIBLE_TOKEN))),
+                                                                HapiParserUtil.asHeadlongAddress(
+                                                                        asAddress(
+                                                                                spec.registry()
+                                                                                        .getAccountID(
+                                                                                                OWNER))),
+                                                                HapiParserUtil.asHeadlongAddress(
+                                                                        asAddress(
+                                                                                spec.registry()
+                                                                                        .getAccountID(
+                                                                                                RECIPIENT))),
+                                                                BigInteger.TWO)
+                                                        .gas(500_000L)
+                                                        .via(TRANSFER_FROM_ACCOUNT_TXN)
+                                                        .hasKnownStatus(CONTRACT_REVERT_EXECUTED))))
```

Comment:
Q: are we able to additionally check the revert reason in this case?

Reply:
We can --- [example here](https://github.com/hashgraph/hedera-services/blob/6e39a60fd82ac9750cc92c7cf0e7ebf521eb9c1c/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/leaky/LeakyContractTestsSuite.java#L285-L291). Agreed that we should probably have such an assertion.

Reply:
yeah did not do that assertion as there is no child record set.

Reply:
@Neeharika-Sompalli do we need a test with a child transaction or is there already coverage in another test for that?

Reply:
@Nana-EC In this case there is no child record exported. There will be `CONTRACT_REVERT_EXECUTED` on top level record.

Reply:
Understood, logic precedes precompile execution. Gotcha